backend/src/main/dto/PayDTOs.ts

```diff
@@ -0,0 +1,20 @@
+export interface PayCreationDTO {
+  issueDate: Date;
+  hoursWorked: number;
+  status?: PayStatus;
```

Comment:
Why is there a "?" here?

Reply:
I added as optional because the default value is `NOT_PAID` so if the user doesn't provide it, it is fine.